README.md

```diff
@@ -12,16 +12,34 @@ Support is guaranteed for Python 2.7.X only at this time.  It has not been teste
 
 ### Installing
 
+#### Dependencies
+
+The following dependencies are required for robotframework-pykafka.
+
+```
+pip install [-U] [--user] \
```

Comment:
This should be `pip install -r requirements.txt`

The dependencies with python are typically deployed that way.